user/serializers.py

```diff
@@ -0,0 +1,25 @@
+from rest_framework import serializers
+from django.contrib.auth import password_validation as validators
+
+from .models import CustomUser
+
+class UserSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CustomUser
```

Comment:
I can't see any use of serializer in views, So what's the point of creating these?